bullseye/etc/entry.sh

```diff
@@ -33,28 +33,30 @@ fi
 
 # Rewrite Config Files
 
-sed -i -e "s/{{SERVER_HOSTNAME}}/${CS2_SERVERNAME}/g" \
-       -e "s/{{SERVER_CHEATS}}/${CS2_CHEATS}/g" \
-       -e "s/{{SERVER_HIBERNATE}}/${CS2_SERVER_HIBERNATE}/g" \
-       -e "s/{{SERVER_PW}}/${CS2_PW}/g" \
-       -e "s/{{SERVER_RCON_PW}}/${CS2_RCONPW}/g" \
-       -e "s/{{TV_ENABLE}}/${TV_ENABLE}/g" \
-       -e "s/{{TV_PORT}}/${TV_PORT}/g" \
-       -e "s/{{TV_AUTORECORD}}/${TV_AUTORECORD}/g" \
-       -e "s/{{TV_PW}}/${TV_PW}/g" \
-       -e "s/{{TV_RELAY_PW}}/${TV_RELAY_PW}/g" \
-       -e "s/{{TV_MAXRATE}}/${TV_MAXRATE}/g" \
-       -e "s/{{TV_DELAY}}/${TV_DELAY}/g" \
+sed -r -i -e "s/^(hostname) .*/\1 ${CS2_SERVERNAME}/g" \
```

Comment:
I don't think the sed changes to server.cfg are necessary as server.cfg is overwritten with the blank template on every container start